### DISCUSSION TITLE
Cikkek

### DISCUSSION THREAD

**Discussion by qbeer** [csabaiBio/ai4covid#5] [Show and tell]
Elsődleges cikk a leírásból: https://www.sciencedirect.com/science/article/pii/S1361841521002619

**Comment by abiricz** [comment]
Találtam egy ilyet: https://elifesciences.org/articles/70640. Adatokat nem tehetnek közzé, de a cikk elolvasható. Nem csináltak nagy varázslatot.

**Comment by icsabai** [comment]
Lefuttattam a PCA-s módszert a nemek jóslására (0.7-es accuracy-t ér el), Így néz ki a súlyozott kép:
![image](https://user-images.githubusercontent.com/18164480/156651175-a1180376-09d9-4f4d-882e-4d3db2b70f72.png)

Összehasonlításul ezt találtam:
- https://www.researchgate.net/publication/221506231_Mining_Lung_Shape_from_X-Ray_Images
a Fig. 6 jobb fölső része mutatja, hogy összeregisztrált férfi és női tüdők közt ni a különbség. Szerintem bele lehet látni.
